monkey/monkey_island/cc/ui/src/components/ui-components/InfoPane.js

```diff
@@ -40,10 +41,13 @@ function getTitle(props) {
 
 function getLinkButton(props) {
   if (typeof (props.link) == 'string') {
+    const sanitizedLink = sanitizeURI(props.link);
+
     return (
-      <Button variant={'link'} className={'pane-link'} href={props.link} target={'_blank'}>
-        <FontAwesomeIcon icon={faQuestionCircle}/>
-      </Button>)
+      <Button variant={'link'} className={'pane-link'} href={sanitizedLink} target='_blank'>
```

Comment:
Are there any hooks or maybe we can override the Button to always sanitize the href?

Reply:
We can. That being said, sanitization is only required for user inputs which are used only in this component.
I think that due to the fact we're facing a major refactor anyway, we can just use it as it is now and keep it in mind for the refactoring.

Reply:
Sounds good to me.